common/src/main/java/com/wynntils/services/mapdata/MapDataService.java

```diff
@@ -125,6 +127,16 @@ public MapIcon getIconOrFallback(MapFeature feature) {
                         .get());
     }
 
+    public Optional<MapAttributes> getOverrideAttributesForFeature(MapFeature feature) {
+        return overrideProviders.values().stream()
+                .filter(attr -> attr.getOverridenFeatureIds()
+                                .anyMatch(attrFeatureId -> attrFeatureId.equals(feature.getFeatureId()))
+                        || attr.getOverridenCategoryIds()
+                                .anyMatch(attrCategoryId -> attrCategoryId.equals(feature.getCategoryId())))
+                .map(MapDataOverrideProvider::getOverrideAttributes)
+                .findFirst();
+    }
```

Comment:
This logic is slightly flawed, as it assumes that only 1 override is active per feature, whereas it is completely fine logically, if multiple overrides are present. I'll fix this now.